lib/step_flow/workflows/workflows.ex

```diff
@@ -191,8 +147,8 @@ defmodule StepFlow.Workflows do
         query
 
       date_value ->
-        date = Date.from_iso8601!(date_value)
-        from(workflow in query, where: fragment("?::date", workflow.inserted_at) <= ^date)
+        date = NaiveDateTime.from_iso8601!(date_value)
```

Comment:
to keep retro-compatibility, you may need to do:
```elixir
date =
  case NaiveDateTime.from_iso8601!(date_value) do
    {:ok, date} -> date
   _ -> Date.from_iso8601!(date_value)
  end
```

Reply:
what if I check all usage of the endpoint using the function in the backend?